lib/parse-request.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const url = require('url');
+const queryString = require('querystring');
+
+// const keywords = ['id'];
+
+//FOR later
```

Comment:
I would just do this in a different commit rather than having commented out code .